assets/scripts/app/initialization.js

```diff
@@ -1,16 +1,209 @@
-/* global debug, system, app, ENV */
+/* global debug, system, app, ENV, CustomEvent, imagesToBeLoaded */
```

Comment:
Similar to comments elsewhere, let's use `window.CustomEvent` to refer to browser-provided globals instead of declaring it here. Also, can we change `imagesToBeLoaded` to be an import from `load_resources`?